doc/requirements.txt

```diff
@@ -2,7 +2,7 @@
 # used by Read The Docs to install python required
 # modules with pip.
 
-sphinx
+sphinx<7
```

Comment:
Good to comment why we need spinx < 7 (breaking change in 7?  rtd them not supported in 7+ ?)